### ISSUE TITLE
Easiest way to add audio to sequence level AAF's?

### ISSUE THREAD

**Issue by kevinparkbss** [markreidvfx/pyaaf2#71]
Hi Mark!

What would be the easiest way to include an audio track that is attached to each video clip to a sequence level AAF? At the moment, I am importing the audio essence to a mastermob in addition to importing the dnxhd essence. This gives me the sourceclip with a video and audio track in the avid bin but after adding them to a sequence on the CompositionMob slot, the resulting sequence AAF only has a video track and no audio. I'm assuming that I need to create an audio sequence in the CompositionMob as well but I'm unsure of when to do that and how to ensure that the video and audio are linked in the sequence in the Avid. Do I need to add the mastermob sourceclip to both a video and audio sequence, in a sense iterating through the shot and filler list twice? If I do this, I can create a sequence AAF with a video track and an audio track but the video and audio for a source comes in as 2 separate pieces of media and move in the timeline independently.

bottom line is that I would like to place clips with both video and audio into a sequence level AAF with filler. And I would like these to be from a single source clip so that they move on the timeline together. If I create a single shot AAF and not a sequence AAF, the clip in the bin has both video and audio but not if I create a sequence AAF.

Thanks!

**Comment by markreidvfx** [comment]
I think I follow. Unfortunately it can be quite complicated. Yes you need to create sourceclips that line up for both the video and audio tracks from the same single MasterMob. I'm not aware of any other info that may be required to link them together, but maybe avid needs something. Do you have a script you can share or a aaf?
